### PR TITLE
upgrading repo to use block-slots >=1.0.0

### PR DESCRIPTION
**Pull request by oadeojo**
#MAJOR#

upgrading repo to use block-slots >=1.0.0

### CODE REVIEW

**Comment by sglanzer** [comment]
Approved

File: addon/templates/components/frost-info-bar.hbs [hunk 1]
```diff
@@ -1,31 +1,41 @@
...
 {{/yield-slot}}
 
-{{#yield-slot 'actions'}}
-    <div class="action">
-      {{yield (block-params (hash
-        button=(component 'frost-button'
-          priority='tertiary'
-          size='medium'
-          vertical=true)))}}
+{{#yield-slot 'actions' (block-params
```
Comment (rox163) [inline addon/templates/components/frost-info-bar.hbs]: curious as to why the buttons are outside the actions div?

File: package.json [hunk 2]
```diff
@@ -35,19 +36,22 @@
     "ember-cli-mocha": "0.10.1",
     "ember-cli-notifications": "3.2.0",
     "ember-cli-uglify": "1.2.0",
+    "ember-computed-decorators": "0.2.2",
     "ember-data": "^2.4.0",
     "ember-disable-proxy-controllers": "1.0.1",
     "ember-export-application-global": "1.0.5",
-    "ember-frost-core": ">=0.0.14 <2.0.0",
+    "ember-frost-core": "0.17.3",
```
Comment (rox163) [inline package.json]: should update core to at least 0.20.0 so you can  implement the ember-hook stuff here in the future.

File: tests/dummy/app/pods/demo/template.hbs [hunk 3]
```diff
@@ -1,14 +1,49 @@
 {{#frost-info-bar as |slot|}}
-  {{#block-slot slot 'title'}}
-      User accounts
+  {{#block-slot slot 'icon'}}
```
Comment (rox163) [inline tests/dummy/app/pods/demo/template.hbs]: is the word 'slot' still needed here?

File: addon/templates/components/frost-info-bar.hbs (hunk 1 above)
Comment (sglanzer) [inline addon/templates/components/frost-info-bar.hbs]: @rox163 that's a parameter hash for the yield

File: addon/styles/addon.scss [hunk 4]
```diff
@@ -1,36 +1,59 @@
...
-  height: $info-bar-height + $info-bar-border-width;
-  box-sizing: border-box;
+  height: $info-bar-height;
   border-bottom: $info-bar-border-width solid $frost-color-lgrey-1;
   background-color: $frost-color-white;
   padding: $info-bar-padding-top-bottom $info-bar-padding-right $info-bar-padding-top-bottom $info-bar-padding-left;
 
-  .title {
-    line-height: 1;
+  .icon {
```
Comment (sglanzer) [inline addon/styles/addon.scss]: We should update all of the `.icon` type classes to something like `.frost-info-bar-icon`

File: tests/dummy/app/pods/demo/template.hbs (hunk 3 above)
Comment (sglanzer) [inline tests/dummy/app/pods/demo/template.hbs]: Nope

Comment (sglanzer) [inline tests/dummy/app/pods/demo/template.hbs]: We should also wrap this in an ember-code-snippet block for the demo

**Pull request merged**